docs/reference/images.md

```
@@ -96,7 +103,7 @@ The `base_rootfs` field is used when no `--image` flag is specified. The `images
 Create a shed with a specific variant:
 
 ```bash
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)